lightkurve/search.py

```diff
@@ -231,6 +250,53 @@ def _default_download_dir(self):
 
         return download_dir
 
+    def _fetch_tesscut_path(self, target, download_dir, cutout_size):
+        """Downloads TESS FFI cutout and returns path to local file.
+
+        Parameters
+        ----------
+        download_dir : str
+            Path to location of `.lightkurve-cache` directory where downloaded
+            cutouts are stored
+        cutout_size : int or float
+            Side length of cutout in pixels
+
+        Returns
+        -------
+        path : str
+            Path to locally downloaded cutout file
+        """
+        from astroquery.mast import TesscutClass
+        tc = TesscutClass()
+
+        # Resolve SkyCoord of given target
+        coords = self._resolve_coords(target)
+        sector = int(self.table[0]['description'][-2])
+        if cutout_size is None:
+            cutout_size = 5
+        cutout_path = tc.download_cutouts(coords, size=cutout_size,
+                                          sector=sector, path=download_dir)
+
+        path = os.path.join(download_dir, cutout_path[0][0])
+        return path
+
+    def _resolve_coords(self, target):
```

Comment:
I propose removing this function because it really only wraps one line, i.e. `MastClass()._resolve_object(target)`.  It'll keep our code shorter if we call that line directly where needed.